### ISSUE TITLE
Angular 7 Support

### ISSUE THREAD

**Issue by danielmoncada** [HackedByChinese/ng2-idle#119]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[X ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/HackedByChinese/ng2-idle/blob/master/CONTRIBUTING.md#getting-help
```

**Current behavior**

While using in an Angular 7 project, I get the following warnings:
```cs
warning " > @ng-idle/core@6.0.0-beta.3" has incorrect peer dependency "@angular/common@^6.0.0".
warning " > @ng-idle/core@6.0.0-beta.3" has incorrect peer dependency "@angular/core@^6.0.0".
```

**Expected behavior**
no warnings

**Comment by xinthose** [comment]
The error I get with Angular 7 is this:
```
ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'idle' of undefined
TypeError: Cannot read property 'idle' of undefined
```

**Comment by anveshreddy48** [comment]
+1

**Comment by r-guerin** [comment]
+1

**Comment by HackedByChinese** [comment]
This is available as @ng-idle/core@7.0.0-beta.1 and @ng-idle/keepalive@7.0.0-beta.1
